sec-keys/openpgp-keys-apache-xalan-c/openpgp-keys-apache-xalan-c-1.12.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="OpenPGP keys used to sign Apache Xalan-C"
+HOMEPAGE="https://xalan.apache.org/"
+SRC_URI="https://github.com/apache/xalan-c/releases/download/Xalan-C_$(ver_rs 1-2 _)/KEYS -> ${P}"
```

Comment:
SRC_URI="https://downloads.apache.org/xalan/xalan-c/KEYS -> ${P}"

I'd prefer the KEYS from apache, not from a mirror.

Reply:
My logic was not to get both the tarball and the keys from the same place, one would need to crack 2 servers to replace them.

Reply:
> My logic was not to get both the tarball and the keys from the same place, one would need to crack 2 servers to replace them.

Understand.  Tarballs usually com from the mirrors. Compare to https://github.com/gentoo/gentoo/pull/27735

Reply:
Good.  I'll change the links then :+1: